extensions/uploadthing/src/lib/hooks.tsx

```diff
@@ -93,3 +94,16 @@ export const useAppInfo = () => {
 
   return query.data;
 };
+
+export const useFiles = () => {
+  const utapi = new UTApi({ token: getToken() });
+  const { isLoading, data } = useCachedPromise(
+    async () => {
+      const res = await utapi.listFiles();
```

Comment:
may want to paginate this, by default it returns first 500

possibly want a  lower limit to start, but also likely would want to have a paginated scroll if possible